src/pkg/artifactrash/manager.go

```diff
@@ -32,9 +33,9 @@ type Manager interface {
 	// Delete ...
 	Delete(ctx context.Context, id int64) (err error)
 	// Filter ...
-	Filter(ctx context.Context) (arts []model.ArtifactTrash, err error)
+	Filter(ctx context.Context, timeWindow int64) (arts []model.ArtifactTrash, err error)
```

Comment:
We should clarify the window here is hour

Reply:
updated.